src/main/java/kitchenpos/application/TableService.java

```diff
@@ -34,41 +32,26 @@ public List<OrderTable> list() {
     }
 
     @Transactional
-    public OrderTable changeEmpty(final Long orderTableId, final OrderTable orderTable) {
+    public OrderTable changeEmpty(Long orderTableId, boolean isEmpty) {
         final OrderTable savedOrderTable = orderTableDao.findById(orderTableId)
                 .orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
    @Transactional
    public OrderTable changeEmpty(Long orderTableId, boolean isEmpty) {
        final OrderTable savedOrderTable = findOrderTableById(orderTableId);
        validateOrderStatus(orderTableId);
        savedOrderTable.changeEmpty(isEmpty);
        return orderTableDao.save(savedOrderTable);
    }

    private OrderTable findOrderTableById(final Long orderTableId) {
        return orderTableDao.findById(orderTableId)
                .orElseThrow(IllegalArgumentException::new);
    }
```
밑에서도 사용되고 있는데 orElseThrow()가 반복되고 있어서 이렇게 메소드 추출을 해주는 것도 괜찮을 것 같네요 !